src/lib/structures/Josh.ts

```diff
@@ -639,7 +674,12 @@ export class Josh<StoredValue = unknown> {
 
     this.runBehaviorOnPayloadError(payload);
 
-    if (isPayloadWithData<Record<string, StoredValue>>(payload)) return this.convertBulkData(payload.data, returnBulkType);
+    if (isPayloadWithData<Record<string, StoredValue>>(payload)) {
+      return this.convertBulkData(
+        payload.data,
+        isPrimitive(valueOrReturnBulkType) || typeof valueOrReturnBulkType === 'symbol' ? returnBulkType : valueOrReturnBulkType || returnBulkType
```

Comment:
bulk is not a symbol

Reply:
Hence why we check, and if so, don't use valueOrReturnBulkType

Reply:
Why would someone pass a symbol? Where in the typings does this method allow for one?